source/_integrations/vicare.markdown

```diff
@@ -34,17 +34,16 @@ There is currently support for the following device types within Home Assistant:
 
 {% include integrations/config_flow.md %}
 
-Set `username`and `password` to your Viessmann Developer Portal login credentials.
-The required Client ID can be obtained as follows:
-1. Register and login in the [Viessmann Developer Portal](https://developer.viessmann.com). Use the same account as the one you registered your device with in ViCare.
-2. In the menu navigate to API Keys.
-3. Create a new OAuth client using the following data:
-  ```txt
-  Name: PyViCare
-  Google reCAPTCHA: Disabled
-  Redirect URIs: vicare://oauth-callback/everest
-  ```
-4. Copy the Client ID from the Viessmann portal and enter this in the API Key field in Home Assistant.
+Use your ViCare app login credentials for *username* and *password*.
+
+The required *client ID* can be obtained from the [Viessmann Developer Portal](https://app.developer.viessmann.com/):
+1. Login with **your existing ViCare app username/password**.
+2. On the developer dashboard click *add* in the *clients* section.
```

Comment:
minor, but I'd probably add a comma here:
```suggestion
2. On the developer dashboard, click *add* in the *clients* section.
```

---

Also, looking at the [Google Assistant docs](https://www.home-assistant.io/integrations/google_assistant/#google-cloud-platform-configuration) as an example, bold text is used for the "buttons" (rather than italic). They're also capitalized. So, this seems like it would be a better fit:

```suggestion
2. On the developer dashboard, click **Add** in the **Clients** section.
```